packages/website/src/snippets.ts

```diff
@@ -150,10 +132,10 @@ export const touchpointUiSetupSnippet = ({
       ? `
   theme: ${JSON.stringify(theme)}`
       : ""
-  }${
-    renderCustomModalitiesExample
+  },${
+    templateComponents === "museumComponents"
       ? `
-  customModalities: { REPLACE_WITH_CUSTOM_MODALITIES }`
+  customModalities: { MuseumExhibitDetails, MuseumExhibitCarousel }`
```

Comment:
This series of `? true : false` is just hard to read. 

I think prettier is doing the opposite for me here lol